content/en/tracing/profiling/getting_started.md

```diff
@@ -0,0 +1,209 @@
+    ```
+
+     **Note**: Profiling is available in the `dd-java-agent.jar` library in versions 0.44+.
+
+3. Set `-Ddd.profiling.enabled` flag or `DD_PROFILING_ENABLED` environment variable to `true`. Update to your service invocation should look like:
+
+    ```diff
+    java -javaagent:dd-java-agent.jar -Ddd.profiling.enabled=true -jar <YOUR_SERVICE>.jar <YOUR_SERVICE_FLAGS>
+    ```
+
+**Note**:
+
+- The `-javaagent` needs to be run before the `-jar` file, adding it as a JVM option, not as an application argument. For more information, see the [Oracle documentation][3]:
+
+    ```shell
+    # Good:
+    java -javaagent:dd-java-agent.jar ... -jar my-service.jar -more-flags
+    # Bad:
+    java -jar my-service.jar -javaagent:dd-java-agent.jar ...
+    ```
+
+- It is strongly recommended to add tags like `service` or `version` as it provides the ability to slice and dice your profiles across these dimensions, enhancing your overall product experience. Use environment variables to set the parameters:
+
+| Environment variable                             | Type          | Description                                                                                      |
+| ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
+| `DD_PROFILING_ENABLED`                           | Boolean       | Alternate for `-Ddd.profiling.enabled` argument. Set to `true` to enable profiling.               |
+| `DD_SERVICE`                                     | String        | The Datadog [service][2] name.     |
+| `DD_ENV`                                         | String        | The Datadog [environment][4] name, for example `production`.|
+| `DD_VERSION`                                     | String        | The version of your application.                             |
+| `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.  |
+
+
+[1]: https://docs.oracle.com/javacomponents/jmc-5-4/jfr-runtime-guide/about.htm
+[2]: https://app.datadoghq.com/account/settings#agent/overview
+[3]: https://docs.oracle.com/javase/7/docs/technotes/tools/solaris/java.html
+[4]: https://github.com/DataDog/dd-trace-java/releases/tag/v0.55.1
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python 3.5+. To begin profiling applications:
+
+1. If you are already using Datadog, upgrade your agent to version [7.20.2][1] or [6.20.2][1].
+
+2. Install `ddtrace` which contains both tracing and profiling:
+
+    ```shell
+    pip install ddtrace
+    ```
+
+     **Note**: Profiling is available in the `ddtrace` library for versions 0.36+.
```

Comment:
Maybe something about removing `[profile]` if upgrading from a previous version?

Reply:
If we knew it was a clean install we wouldn't need to specify a min version.